src/lib/utils.ts

```diff
@@ -7,6 +7,13 @@ import twemojiModule from "twemoji";
 import { base } from "@app/lib/router";
 import { config } from "@app/lib/config";
 
+export interface Item<T> {
```

Comment:
Shouldn't this live in the module context of `CobStateButton`?

Reply:
We use it also in `Dropdown` that's why I didn't wanted to move it to `CobStateButton`

Reply:
Ok, in that case it should live in the `Dropdown` component's module context. Since `CobStateButton` uses `Dropdown`.

Reply:
Done in 7c70473